test/server/graphql/v2/mutation/CreateCollectiveMutations.test.js

```diff
@@ -139,160 +119,4 @@ describe('server/graphql/v2/mutation/CreateCollectiveMutations', () => {
       expect(memberInvitationActivities[1].data.memberCollective.name).to.eq('Another admin');
     });
   });
-
-  describe('with GitHub repository', async () => {
```

Comment:
@znarf : Removing these as we are just showing error and letting the the application process so no failure to test.

Reply:
Ok, let's review the implications of this quietly before merging.

Reply:
@znarf : What do you mean by "review the implications quietly"? Do I need to do anything on my end? 🙂